src/classes/CanvasRenderingContext2D.js

```diff
@@ -1282,22 +1271,26 @@ export default class CanvasRenderingContext2D {
 
   set shadowColor(value) {
     if (typeof value === 'string') {
-      const result = parseCSSColor(value);
-
-      if (result) {
-        this._shadowColorStack[this._stackIndex] = result;
-        const event = createCanvasEvent(
-          'shadowColor',
-          getTransformSlice(this),
-          { value: result },
-        );
-        this._events.push(event);
-      }
+      try { 
+        value = new MooColor(value);
+        this._shadowColorStack[this._stackIndex] = value;
+      } catch (e) { return; }
+
+      const event = createCanvasEvent(
+        'shadowColor',
+        getTransformSlice(this),
+        { value },
+      );
+      this._events.push(event);
     }
   }
 
   get shadowColor() {
-    return this._shadowColorStack[this._stackIndex];
+    const color = this._shadowColorStack[this._stackIndex];
```

Comment:
We should be able to return the `shadowColorStack` value here just like above comments

Reply:
uses the new helper function `serializeColor`